src/text/helper.ts

```diff
@@ -77,6 +77,18 @@ const getFontWeight = (weight: TextWeight) => {
 // =============================================================================
 // TEXT STYLE
 // =============================================================================
+const getMaxLinesLineStyle = (maxLines: number | undefined = undefined) => {
```

Comment:
the internal function can be more explicit about the number of args required

```suggestion
const getMaxLinesLineStyle = (maxLines: number | undefined) => {
```